pype/hosts/tvpaint/hooks/pre_launch_args.py

```diff
@@ -78,7 +78,7 @@ def workfile_path(self):
             # TODO add ability to set different template workfile path via
             # settings
             pype_dir = os.path.dirname(os.path.abspath(tvpaint.__file__))
-            template_path = os.path.join(pype_dir, "template.tvpp")
+            template_path = os.path.join(pype_dir, "api", "template.tvpp")
```

Comment:
I'd move the template directly to host directory, rather than inside API. or possibly, `resources` inside the host?

Reply:
Done